models/aeif_cond_beta_multisynapse.h

```diff
@@ -74,7 +74,9 @@
  the synapse i is excitatory or inhibitory depending on the value of E_{rev,i}
  and the differential equation for the spike-adaptation current w is:
 
- tau_w * dw/dt = a(V - E_L) - W
+ tau_w * dw/dt = a(V - E_L) - w
+
+ When the neuron fires a spike, the adaptation current w <- w + b.
```

Comment:
Inserting "undergoes a step increment of value b" before the formula would make the sentence less abrupt (and possibly clearer (to me).